docs/userguide/labels-custom-metadata.md

```diff
@@ -1,230 +1,106 @@
 <!--[metadata]>
 +++
-title = "Apply custom metadata"
-description = "Learn how to work with custom metadata in Docker, using labels."
-keywords = ["Usage, user guide, labels, metadata, docker, documentation, examples,  annotating"]
+title = "Managing Docker Object Labels"
```

Comment:
Yes, we standardize on sentence capitalization; see the style guide; https://docs.docker.com/opensource/doc-style/#/in-headings